### PR TITLE
update download overview to vbt

### LINKED ISSUE #1620: /download/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* converted /download to VBT

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- see the page http://0.0.0.0:8001/desktop/developers
- [see the demo](http://www.ubuntu.com-1620-desktop-overview-vbt.demo.haus/download)

## Issue / Card

Fixes #1620

### CODE REVIEW

File: templates/download/index.html [hunk 1]
```diff
@@ -1,104 +1,111 @@
...
-            <div class="six-col last-col box equal-height">
-                <div class="four-col">
-                    <h2><a href="/download/cloud">Ubuntu Cloud&nbsp;&rsaquo;</a></h2>
-                    <p>Ubuntu is the reference OS for OpenStack. Canonical&rsquo;s OpenStack Autopilot is a fully automated deployment of an OpenStack cloud on Ubuntu &mdash; just add servers.</p>
-                </div>
-                <div class="two-col last-col align-center equal-height__item equal-height__align-vertically">
-                    <a href="/download/cloud"><img src="{{ ASSET_SERVER_URL }}dfd42685-picto-cloud-darkaubergine.svg" alt="" width="113" height="113" /></a>
-                </div>
-            </div>
+<div class="p-strip">
```
Comment (barrymcgee) [inline templates/download/index.html]: This strip needs `is-deep` according to @yaili's pattern spec - https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-overview.png

File: static/sass/_v1_pattern_card.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@mixin ubuntu-p-card {
+  .p-card__footer--upgrade {
+    background: $color-light url('#{$assets-path}f68488b1-picto-upgrade-warmgrey.svg') 20px center no-repeat;
+    background-size: 40px 40px;
```
Comment (barrymcgee) [inline static/sass/_v1_pattern_card.scss]: We should avoid using pixel values in CSS. Ideally, you should use [Vanilla spacing variables](https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_settings_spacing.scss) - in this case `padding: $sp-xx-large $sp-xx-large`.

Where there is not an exact 1:1 match, choose the nearest value. If that value breaks the design, you may then be justified in using an alternate `rem` value but this is the last resort option.

File: static/sass/_v1_pattern_card.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    padding: 20px 20px 20px 84px;
+    margin: 20px -20px 0 -20px;
+  }
+
+  html.no-svg,
```
Comment (barrymcgee) [inline static/sass/_v1_pattern_card.scss]: These selectors are over-qualified, please remove `html`

**Comment by pmahnke** [comment]
@barrymcgee updated

File: static/sass/_v1_pattern_card.scss (hunk 3 above)
Comment (pmahnke) [inline static/sass/_v1_pattern_card.scss]: done

File: static/sass/_v1_pattern_card.scss (hunk 2 above)
Comment (pmahnke) [inline static/sass/_v1_pattern_card.scss]: tried as much as I could

File: templates/download/index.html (hunk 1 above)
Comment (pmahnke) [inline templates/download/index.html]: done

**Comment by yaili** [comment]
Some comments:
- [ ] The contextual footer is missing external icons
- [ ] Can the Buy Ubuntu Advantage link in the bottom be a neutral button style?

**Comment by pmahnke** [comment]
@yaili I fixed the button; however the external icons are fixed in another PR... the demo has been updated

**Comment by yaili** [comment]
- [ ] I just noticed that the light grey strip inside the Ubuntu Desktop card is not touching the edges, it's about 1px in L screen and it becomes a bit bigger gap as the screen size reduces

**Comment by pmahnke** [comment]
@yaili I fixed it... and I also added some margin on the text bit as without, it seemed too tight?  obviously I can remove

my version
image

no margin version
image

**Comment by yaili** [comment]
The demo link says "no demo here".

**Comment by barrymcgee** [comment]
@yaili Just kicked it - http://www.ubuntu.com-1620-desktop-overview-vbt.demo.haus/

**Pull request merged**